mod/nginx/bbb/learning-dashboard.nginx

```diff
@@ -1,5 +1,5 @@
 location ~ /learning-analytics-dashboard/([0-9a-f]+-[0-9]+)/(.*) {
-    root /var/bigbluebutton/learning-analytics-dashboard/;
+    root /var/bigbluebutton/;
```

Comment:
I'm not so familiar with nginx configs, but does this safely not provide any way for malicious people to access other files in `/var/bigbluebutton/`?